### PR TITLE
tests in StaffService class

### PR DESCRIPTION
**Pull request by rdmoraes**
Hi guys, I'll make small PR in order to you have less code for reviewing! 
Here I have test cases for StaffService class and small refactoring in DataStorage.

I am a bit confused if I should move these tests to TestDataStorage or keep it on the StaffService, what is your opinion on this?

### CODE REVIEW

**Review by dshirt — COMMENTED** [review]
Looks good to me, but I'd wait till Dan approves.

File: src/main/java/employee/com/StaffService.java [hunk 1]
```diff
@@ -2,31 +2,29 @@
 
 public class StaffService extends EmployeeService {
 
-    StaffService() {
+    public StaffService() {
         employeeDataStoreService = new EmployeeDataStoreService();
         validationService = new ValidationService();
```
Comment (daniel-fitzg) [inline src/main/java/employee/com/StaffService.java:7]: It is for another PR, but the ValidationService class can be deleted now and the declarations in StaffService and ManagerService.

File: PipelineProject.iml [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.junit.jupiter:junit-jupiter-api:5.4.2" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.apiguardian:apiguardian-api:1.0.0" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.opentest4j:opentest4j:1.1.1" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.junit.platform:junit-platform-commons:1.4.2" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.junit.jupiter:junit-jupiter-engine:5.4.2" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.junit.platform:junit-platform-engine:1.4.2" level="project" />
+  </component>
+</module>
```
Comment (daniel-fitzg) [inline PipelineProject.iml]: What is this change? If it is merged will it cause problems for myself or John when we pull from master?

**Review by daniel-fitzg — COMMENTED** [review]
Looks great @rdmoraes Just wondering about the PiplelineProject.iml file?

File: PipelineProject.iml (hunk 2 above)
Comment (rdmoraes) [reply PipelineProject.iml]: I'll add it to .gitignore, it seems to be an autogenerated file in my local Intellij

**Review by daniel-fitzg — APPROVED** [review]
Great job.

**Pull request merged**